packages/@lwc/engine-server/src/__tests__/fixtures/read-only-props/function/modules/x/component/component.js

```diff
@@ -0,0 +1,9 @@
+import { LightningElement, readonly } from 'lwc';
+
+export default class extends LightningElement {
+    foo = { willSays: 'mobs 4eva' };
+    readonlyFoo = readonly(this.foo);
```

Comment:
```suggestion
    readonlyFoo = readonly(this.foo);
    readonlyFoo.mutated = true;
    foo.johnSays = 'arrrgh, me knickers';
```
1. We should try to mutate the readonly proxy to validate that it's readonly
2. We should mutate the original to validate that the readonly proxy is also updated

Reply:
good idea